modules/widgets/wordpress-post-widget.php

```diff
@@ -238,14 +822,59 @@ public function form( $instance ) {
 		</p>
 
 		<?php
+
+		/**
+		 * Show error messages.
+		 */
+		if ( ! empty( $update_errors['message'] ) ) {
+
+			/**
+			 * Prepare the error messages.
+			 */
+
+			$where_message = __( 'An error occurred while downloading ', 'jetpack' );
```

Comment:
I updated the code relating to that in 9a154e831b5823626dc787b780607b06a84a5579. Is this a better way to handle such situation?